### PR TITLE
Remove name or id functionality from resources

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: cloudca/resource_tier.go [hunk 1]
```diff
@@ -46,22 +46,22 @@ func resourceCloudcaTier() *schema.Resource {
...
 				Required:    true,
 				ForceNew:    true,
 				Description: `The network offering name or id (e.g. "Standard Tier" or "Load Balanced Tier")`,
 			},
-			"network_acl": &schema.Schema{
+			"network_acl_id": &schema.Schema{
 				Type:        schema.TypeString,
 				Required:    true,
-				Description: "The network ACL name or id",
+				Description: "Id of the network ACL name",
```
Comment (simongodard) [inline cloudca/resource_tier.go]: Id of the name?

File: cloudca/resource_cloudca_instance.go [hunk 2]
```diff
@@ -56,14 +56,11 @@ func resourceCloudcaInstance() *schema.Resource {
 				},
 			},
 
-			"network": &schema.Schema{
+			"network_id": &schema.Schema{
```
Comment (pdube) [inline cloudca/resource_cloudca_instance.go]: Should we call this tierId?

**Comment by pdube** [comment]
👍

**Comment by simongodard** [comment]
LGTM

File: cloudca/resource_tier.go (hunk 1 above)
Comment (franzpgarcia) [inline cloudca/resource_tier.go]: Will fix

File: cloudca/resource_cloudca_instance.go (hunk 2 above)
Comment (franzpgarcia) [inline cloudca/resource_cloudca_instance.go]: I don't mind. However, the API returns network id could be confusing

**Pull request merged**